Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -28,7 +28,9 @@ protocol BolusEntryViewModelDelegate: AnyObject {
 
     func storeBolusDosingDecision(_ bolusDosingDecision: BolusDosingDecision, withDate date: Date)
     
-    func enactBolus(units: Double, at startDate: Date, completion: @escaping (_ error: Error?) -> Void)
+    func enactBolus(units: Double, automatic: Bool, completion: @escaping (_ error: Error?) -> Void)
+    
+    func logOutsideInsulinDose(startDate: Date, units: Double, insulinType: InsulinType?)
```

Comment:
`logOutsideInsulinDose` -> `storeOutsideInsulinDose`

Reply:
changed with the update to manuallyEntered